knowledge-base/dropdownlist-custom-sorting-order-groups.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Custom Sorting Order for Groups in DropDownList
+description: Learn how to apply a custom order to groups in the DropDownList for Blazor.
+type: how-to
+page_title: How to Set a Custom Grouping Order in Blazor DropDownList
+slug: dropdownlist-custom-sorting-order-groups
+tags: dropdownlist, blazor, telerik, grouping, custom order
+res_type: kb
+ticketid: 1666981, 1579415
+---
+
+## Environment
+
+<table>
+<tbody>
+<tr>
+<td>Product</td>
+<td>DropDownList for Blazor</td>
```

Comment:
```suggestion
<td>DropDownList for Blazor</td>
```
Is this approach applicable to the rest of the selects (AutoComplete, ComboBox etc.)? If so, we can list them here.